employees/tests/test_unit_custom_mixins.py

```diff
@@ -14,33 +18,64 @@
 
 class MonthNavigationMixinCustomMethodsTests(TestCase):
     def setUp(self):
+        self.factory = RequestFactory()
         self.mixin = MonthNavigationMixin()
+        self.current_date = datetime.datetime.now().date()
```

Comment:
Please use `django.utils.timezone`.

Reply:
done, just in one place I use datetime because it was needed to create special date.

Reply:
I see you have figures this out :)